instrumentation/opentelemetry-instrumentation-aws-sdk/src/AwsSdkInstrumentation.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
```

Comment:
Can you please add the strict type declaration `declare(strict_types=1);` and the namespace here?